packages/components/src/chart/d3chart/style.scss

```diff
@@ -1,5 +1,140 @@
 /** @format */
 
+.d3-chart__container {
```

Comment:
Is there any reason to merge `legend.scss` and `style.scss` in the same file? I think both files are quite big already so it's easier to maintain them if they are split.